pyworklistserver/__main__.py

```diff
@@ -111,8 +133,9 @@ def _handle_signal(self, sig, frame):
 
 
 if __name__ == '__main__':
-    server_config, logfile, seedfile, reproduce = _get_serverconfig_from_commandline()
+    server_config, logfile, seedfile, reproduce, configfile = _get_serverconfig_from_commandline()

```

Comment:
See if you can harmonize the name of the user config/worklist config file, so that we have uniform naming of the variable that means the same thing.